src/Font.elm

```diff
@@ -128,6 +152,44 @@ fontStyleProperty isItalic =
     if isItalic then "italic" else "normal"
 
 
+-- ANDROID
+
+type alias AndroidFont =
+    String
+
+android : List Font -> List AndroidFont
+android fonts =
+    groupByFamily fonts
```

Comment:
This is the only place `groupByFamily` is used and we throw away the keys in the next step. `groupByFamily` makes great sense the way it is so no harm, but just worth considering if its return type should be something else.

Reply:
I made use of the `Dict` here purely as a convenient intermediate data structure for building up what is essentially a list of lists.

I'm not sure I understand what you mean about the return type, what would you suggest as an alternative?

Reply:
> I'm not sure I understand what you mean about the return type, what would you suggest as an alternative?

I mean you are returning a datastructure that has keys and values, but only using the values. Returning the values instead would be an example of changing the return type to better fit the use case.

As I said though, the function makes great sense the way it is, so no harm.

Reply:
Oh I see, you mean the return type of `groupByFamily`! Sorry, I thought you meant the return type of `android`. Yeah, good shout.